util/cron/test-perf.chapcs.cc.bash

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+#
+# Run -perflabel cc- performance tests on a chapcs machine with gasnet
+
+CWD=$(cd $(dirname $0) ; pwd)
+
+export CHPL_TEST_PERF_CONFIG_NAME='chapcs'
+
+source $CWD/common-perf.bash
+source $CWD/common-gasnet.bash
+
+export CHPL_NIGHTLY_TEST_CONFIG_NAME="perf.chapcs.cc"
+
+SHORT_NAME=cc
+START_DATE=08/17/17
+
+perf_args="-performance-description $SHORT_NAME -sync-dir-suffix $SHORT_NAME -perflabel cc-"
```

Comment:
What does performance-description do? 'cc' might be too short if it's for something human-readable.